# Keiko Akagi, PhD
**Cancer Genomics | Long-Read Sequencing | Bioinformatics**

I build practical tools at the intersection of **cancer genomics**, **long-read sequencing**, and **virus-induced genomic alteration**

Current focus: robust detection and interpretation of **HPV integration** and **structural variants** in tumors.
I'm also exploring **AI-assisted** scoring and visualization of complex rearrangements.

## Â· About Me
- Interests: long-read (ONT/PacBio), structural variants, virus integration, single-cell/spatial, TCR-seq
- Tooling: Python, R, Snakemake
- Advocate for open science and reproducible research

---
## Â· Featured Repositories
- **[hpv-longread-tools](https://github.com/akagi-genomics/hpv-longread-tools)**
Utilities for HPV long-read analysis: chimeric read detection & SVÂ·BED conversion (MVP).
- *(Coming Soon)* **virus-host-concatemer-resolver**
Resolve & visualize complex virusÂ·host concatemers from long-reads.
- *(Coming Soon)* **single-cell-meta-omics**
Reproducible notebooks for single-cell/spatial tumor heterogeneity meta-analysis


## Â· Collaborations
- **[SVAlignR](https://cran.r-project.org/web/packages/SVAlignR/index.html) SVAlignR: Recovering Structure of Long Molecules from Structural Variation Data developed by [Kevin Coombes](https://github.com/kevincoombes 

<!--
**akagi-genomics/akagi-genomics** is a âœ¨ _special_ âœ¨ repository because its `README.md` (this file) appears on your GitHub profile.

Here are some ideas to get you started:

- ðŸ”­ Iâ€™m currently working on ...
- ðŸŒ± Iâ€™m currently learning ...
- ðŸ‘¯ Iâ€™m looking to collaborate on ...
- ðŸ¤” Iâ€™m looking for help with ...
- ðŸ’¬ Ask me about ...
- ðŸ“« How to reach me: ...
- ðŸ˜„ Pronouns: ...
- âš¡ Fun fact: ...
-->
